apps/sharry/docker-compose.yml

```
@@ -0,0 +1,53 @@
+    image: postgres:16.4-alpine
+    container_name: db
+    volumes:
+      - ${APP_DATA_DIR}/data/postgres/:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_USER=sharry
+      - POSTGRES_PASSWORD=${DB_PASS}
+      - POSTGRES_DB=sharrydb
+    networks:
+      - tipi_main_network
+    labels:
+      runtipi.managed: true
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)